src/main/java/io/zeebe/monitor/zeebe/KafkaStreamsService.java

```diff
@@ -0,0 +1,97 @@
+package io.zeebe.monitor.zeebe;
+
+import io.zeebe.monitor.zeebe.importers.kafka.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+@Component
+public class KafkaStreamsService {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+    @Autowired
+    private KakaProcessAndElementImporter processAndElementImporter;
+
+    @Autowired
+    private KafkaTimerImporter timerImporter;
+
+    @Autowired
+    private KafkaVariableImporter variableImporter;
+
+    @Autowired
+    private KafkaErrorImporter kafkaErrorImporter;
+
+    @Autowired
+    private KafkaMessageImporter kafkaMessageImporter;
+
+    @Autowired
+    private KafkaIncidentImporter kafkaIncidentImporter;
+
+    @Autowired
+    private KafkaMessageSubscriptionImporter kafkaMessageSubscriptionImporter;
+
+    @Autowired
+    private KafkaJobImporter kafkaJobImporter;
+
+    @Autowired
+    private ZeebeKafkaStreams zeebeKafkaStreams;
+
+    @PostConstruct
+    public void start() {
+        zeebeKafkaStreams.kafkaProcessStream().foreach((key, event) -> {
+            if (event != null) {
+                processAndElementImporter.importProcess(event);
+            }
+        });
+
+        zeebeKafkaStreams.kafkaProcessInstanceStream().foreach((key, event) -> {
+            if (event != null) {
+                processAndElementImporter.importProcessInstance(event);
+            }
+        });
+
+        zeebeKafkaStreams.kafkaZeebeTimerStream().foreach((key, event) -> {
+            if (event != null) {
+                timerImporter.importTimer(event);
+            }
+        });
+
+        zeebeKafkaStreams.kafkaZeebeVariableStream().foreach((key, event) -> {
+            if (event != null) {
+                variableImporter.importVariable(event);
+            }
+        });
+
+        zeebeKafkaStreams.kafkaZeebeErrorStream().foreach((key, event) -> {
+            if (event != null) {
+                kafkaErrorImporter.importError(event);
+            }
+        });
+
+        zeebeKafkaStreams.kafkaZeebeMessagreStream().foreach((key, event) -> {
+            if (event != null) {
+                kafkaMessageImporter.importMessage(event);
+            }
+        });
+
+        zeebeKafkaStreams.kafkaZeebeIncidentStream().foreach((key, event) -> {
+            if (event != null) {
+                kafkaIncidentImporter.importIncident(event);
+            }
+        });
+
+        zeebeKafkaStreams.kafkaZeebeMessageSubscriptionStream().foreach((key, event) -> {
+            if (event != null) {
+                kafkaMessageSubscriptionImporter.importMessageSubscription(event);
+            }
+        });
+
+        zeebeKafkaStreams.kafkaZeebeJobStream().foreach((key, event) -> {
+            if (event != null) {
+                kafkaJobImporter.importJob(event);
```

Comment:
okay, I played with it a bit
It looks like you don't new own importers a all
You can build a record just by getting values from `GenericKafkaRecord` and mapping it to a required importer record. 
Below sample is based on `JobRecord`:

`Schema.JobRecord.Builder jobRecordBuilder = Schema.JobRecord.newBuilder();
      jobRecordBuilder.setBpmnProcessId("some value from kafka record");
      //other values
      Schema.JobRecord convertedRecord = jobRecordBuilder.build();
existingJobImporterNotAKafkaSpecificOne.importJob(convertedRecord)
`

Reply:
https://protobuf.dev/getting-started/javatutorial/#builders-messages - some docs